docs/index.rst

```diff
@@ -0,0 +1,20 @@
+.. typing documentation master file, created by
+   sphinx-quickstart on Mon May 24 16:43:52 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to the Python Type System documentation!
+================================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
```

Comment:
Open to thoughts / follow-up PRs for a good initial skeleton TOC to shape the first iteration of docs!

Reply:
How much do we want to steal from mypy docs? https://mypy.readthedocs.io/en/latest/#overview-type-system-reference